src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -3356,22 +3394,39 @@ static void _cloneNode(SvgNode* from, SvgNode* parent, int depth)
 }
 
 
-static void _clonePostponedNodes(Array<SvgNodeIdPair>* cloneNodes, SvgNode* doc)
+static void _clonePostponedNodes(Inlist<SvgNodeIdPair>* cloneNodes, SvgNode* doc)
 {
-    ARRAY_FOREACH(p, *cloneNodes) {
-        auto nodeIdPair = *p;
-        auto defs = _getDefsNode(nodeIdPair.node);
-        auto nodeFrom = _findNodeById(defs, nodeIdPair.id);
-        if (!nodeFrom) nodeFrom = _findNodeById(doc, nodeIdPair.id);
-        if (!_findParentById(nodeIdPair.node, nodeIdPair.id, doc)) {
-            _cloneNode(nodeFrom, nodeIdPair.node, 0);
-            if (nodeFrom && nodeFrom->type == SvgNodeType::Symbol && nodeIdPair.node->type == SvgNodeType::Use) {
-                nodeIdPair.node->node.use.symbol = nodeFrom;
+    auto nodeIdPair = cloneNodes->front();
+    while (nodeIdPair) {
+        auto defs = _getDefsNode(nodeIdPair->node);
+        auto nodeFrom = _findNodeById(defs, nodeIdPair->id);
+        if (!nodeFrom) nodeFrom = _findNodeById(doc, nodeIdPair->id);
+
+        if (!_findParentById(nodeIdPair->node, nodeIdPair->id, doc)) {
+            //Check if none of nodeFrom's children are in the cloneNodes list
+            auto postpone = false;
+            INLIST_FOREACH((*cloneNodes), pair) {
+                if (_checkPostponed(nodeFrom, pair->node, 1)) {
```

Comment:
There are scenarios where `nodeFrom` can be null, please see the example below.

```html
<svg xmlns:xlink="http://www.w3.org/1999/xlink" height="729.7" width="639.65" xmlns="http://www.w3.org/2000/svg">
  <defs>

    <use id="sprite0" height="5.15"  width="4.45" href="#shape0"/>

    <use id="sprite0" height="5.15"  width="4.45" href="#invalidUrl"/>

    <path id="shape0" d="M0 0 h 200 v 200 " fill="#ff00ff" />

  </defs>

  <use height="36.05"  width="31.15" href="#sprite0"/>

</svg>

```

how about this?
```cpp
if (nodeFrom && _checkPostponed(nodeFrom, pair->node, 1)) {
```